caom2-artifact-resolvers/src/main/java/ca/nrc/cadc/caom2/artifact/resolvers/CadcNraoResolver.java

```diff
@@ -0,0 +1,118 @@
+*  software without specific prior      de ce logiciel sans autorisation
+*  written permission.                  préalable et particulière
+*                                       par écrit.
+*                                       
+*  This file is part of the             Ce fichier fait partie du projet
+*  OpenCADC project.                    OpenCADC.
+*                                       
+*  OpenCADC is free software:           OpenCADC est un logiciel libre ;
+*  you can redistribute it and/or       vous pouvez le redistribuer ou le
+*  modify it under the terms of         modifier suivant les termes de
+*  the GNU Affero General Public        la “GNU Affero General Public
+*  License as published by the          License” telle que publiée
+*  Free Software Foundation,            par la Free Software Foundation
+*  either version 3 of the              : soit la version 3 de cette
+*  License, or (at your option)         licence, soit (à votre gré)
+*  any later version.                   toute version ultérieure.
+*                                       
+*  OpenCADC is distributed in the       OpenCADC est distribué
+*  hope that it will be useful,         dans l’espoir qu’il vous
+*  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+*  without even the implied             GARANTIE : sans même la garantie
+*  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+*  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+*  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+*  General Public License for           Générale Publique GNU Affero
+*  more details.                        pour plus de détails.
+*                                       
+*  You should have received             Vous devriez avoir reçu une
+*  a copy of the GNU Affero             copie de la Licence Générale
+*  General Public License along         Publique GNU Affero avec
+*  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+*  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+*                                       <http://www.gnu.org/licenses/>.
+*
+*
+************************************************************************
+*/
+
+
+package ca.nrc.cadc.caom2.artifact.resolvers;
+
+import ca.nrc.cadc.net.StorageResolver;
+import ca.nrc.cadc.net.Traceable;
+import java.net.URI;
+import java.net.URL;
+import org.apache.log4j.Logger;
+
+/**
+ * StorageResolver implementation for the VLASS archive.
+ * This class can convert a VLASS URI into a URL. The conversion is delegated to the AdResolver.
+ *
+ * @author yeunga
+ */
+public class CadcNraoResolver implements StorageResolver, Traceable {
+    public static final String SCHEME = "nrao";
+    protected static final String VLASS_ARCHIVE = "VLASS";
+    private static final Logger log = Logger.getLogger(CadcNraoResolver.class);
+
+    @Override
+    public URL toURL(URI uri) {
```

Comment:
This resolver doesn't need this conditional: just use always use `CadcResolver(SCHEME)` in which case it also doesn't need a special validate because the CadcResolver should do that. 

There is no correct scenario where this resolver gets called with the `cadc` scheme in default `new CadcResolver()` and this extra code will make incorrect config accidentally work.

Reply:
Okay. That means there is no need to handle the "cadc" scheme. Removed relevant code.